playground/frontend/lib/utils/share_code_utils.dart

```diff
@@ -0,0 +1,57 @@
+// ignore_for_file: leading_newlines_in_multiline_strings
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:playground/constants/params.dart';
+
+class ShareCodeUtils {
+  static String currentBaseUrl = Uri.base.toString().split('?')[0];
+
+  static String snippetIdToPlaygroundUrl({required String snippetId}) {
```

Comment:
You may add

```dart
enum PlaygroundView{
  embedded,
  standalone,
}
```

And then add the argument `required PlaygroundView view` to this method. Then use it in `snippetIdToIframeCode`.
This will also improve clarity of the original usage of this method.

Reply:
Done.